Makefile

```diff
@@ -43,12 +45,15 @@ all: apps.img
 	cp apps.img $(FROSTED)/
 
 
-apps.img: $(APPS-y) $(DIR-y)
-	@mkdir -p gdb
-	@mv binutils/out/*.gdb $(PWD)/gdb || true
-	@mv hw-utils/out/*.gdb $(PWD)/gdb || true
-	@mv netutils/out/*.gdb $(PWD)/gdb || true
-	$(FROSTED)/tools/xipfstool $@ $(APPS-y) $(addsuffix /out/*,$(DIR-y))
+apps.img: $(APPS-y) $(DIR-y) $(OUT-y) $(GDB-y)
+	@mkdir -p $(PWD)/lib/out
+	@find binutils/out/* -type f -executable | grep -v "gdb" | xargs cp -t $(OUT) || true 
+	@find netutils/out/* -type f -executable | grep -v "gdb" | xargs cp -t $(OUT) || true 
+	@find netutils/out/* -type f -executable | grep -v "gdb" | xargs cp -t $(OUT) || true
+	@find binutils/out/* -type f -executable | grep "gdb" | xargs cp -t $(GDB) || true 
+	@find hw-utils/out/* -type f -executable | grep "gdb" | xargs cp -t $(GDB) || true 
+	@find netutils/out/* -type f -executable | grep "gdb" | xargs cp -t $(GDB) || true
+	$(FROSTED)/tools/xipfstool $@ $(APPS-y) $(DIR-y)
```

Comment:
Hi , just amended the Makefile, let me know.

2016-03-17 23:39 GMT+01:00 Daniele Lacamera <notifications@github.com>:

> In Makefile
> <https://github.com/insane-adding-machines/frosted-userland/pull/3#discussion_r56589570>
> :
>
> > @@ -43,12 +45,15 @@ all: apps.img
> >  	cp apps.img $(FROSTED)/
> >
> >
> > -apps.img: $(APPS-y) $(DIR-y)
> > -	@mkdir -p gdb
> > -	@mv binutils/out/*.gdb $(PWD)/gdb || true
> > -	@mv hw-utils/out/*.gdb $(PWD)/gdb || true
> > -	@mv netutils/out/*.gdb $(PWD)/gdb || true
> > -	$(FROSTED)/tools/xipfstool $@ $(APPS-y) $(addsuffix /out/*,$(DIR-y))
> > +apps.img: $(APPS-y) $(DIR-y) $(OUT-y) $(GDB-y)
> > +	@mkdir -p $(PWD)/lib/out
> > +	@find binutils/out/* -type f -executable | grep -v "gdb" | xargs cp -t $(OUT) || true
> > +	@find netutils/out/* -type f -executable | grep -v "gdb" | xargs cp -t $(OUT) || true
> > +	@find netutils/out/* -type f -executable | grep -v "gdb" | xargs cp -t $(OUT) || true
> > +	@find binutils/out/* -type f -executable | grep "gdb" | xargs cp -t $(GDB) || true
> > +	@find hw-utils/out/* -type f -executable | grep "gdb" | xargs cp -t $(GDB) || true
> > +	@find netutils/out/* -type f -executable | grep "gdb" | xargs cp -t $(GDB) || true
> > +	$(FROSTED)/tools/xipfstool $@ $(APPS-y) $(DIR-y)
>
> stuff in extra/ should also become part of the xipfs distribution: try
> putting a readme.txt in extra and ensure it gets in the xipfs?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/insane-adding-machines/frosted-userland/pull/3/files/18f01db80bf0d4a6657255b5b8aa3db73e7f77e6#r56589570>
>